app/views/shared/_bank_statements.html.erb

```diff
@@ -0,0 +1,125 @@
+          </label>
+          <p><%= t(".upload_paragraph") %></p>
+          <div class="govuk-hint"><%= t(".size_hint") %></div>
+          <% if @form.errors %>
+            <p id="dropzone-mandatory-error" class="govuk-error-message dropzone-error govuk-!-margin-bottom-1">
+              <% @form.errors.errors.each do |error| %>
+                <span class="govuk-visually-hidden">Error:</span>
+                <%= error.message %>
+              <% end %>
+            </p>
+          <% end %>
+
+          <span class="hidden" aria-hidden="true" id="application-id"><%= @form.legal_aid_application_id %></span>
+          <span class="hidden" aria-hidden="true" id="dropzone-url" data-url="/v1/bank_statements"></span>
+          <p id="dropzone-file-error" class="govuk-error-message dropzone-error hidden">
+            <span class="govuk-visually-hidden">Error:</span>
+          </p>
+          <div class="dropzone" id="dropzone-form">
+            <div class="dz-message" data-dz-message>
+              <p class="govuk-body govuk-!-padding-top-2 script"><%= t(".dropzone_message") %></p>
+              <button id="dz-upload-button" class="govuk-button govuk-button--secondary script" tabindex="0"><%= t(".choose_files_btn") %></button>
+            </div>
+          </div>
+        </div>
+
+        <div class="fallback no-script">
+          <%= form.govuk_file_field :original_file, name: :original_file, label: { text: t("generic.attach_file"), size: "m" },
+                                                    hint: { text: t(".size_hint") },
+                                                    classes: ["govuk-file-upload moj-multi-file-upload__input"] %>
+          <%= form.govuk_submit(
+                t("generic.upload"),
+                id: "upload",
+                name: "upload_button",
+                value: "Upload",
+                secondary: true,
+              ) %>
+        </div>
+      </div>
+
+      <div aria-live="assertive" class="govuk-visually-hidden" id="file-upload-status-message"></div>
+
+      <% if @successfully_deleted %>
+        <span role="alert" class="govuk-visually-hidden"><%= @successfully_deleted %></span>
+      <% end %>
+      <% if @error_message || @successful_upload %>
+        <span role="alert" class="govuk-visually-hidden"><%= @successful_upload %><%= @error_message %></span>
+      <% end %>
+
+    <% end %>
+  <% end %>
+
+  <h2 class="govuk-heading-m"><%= t(".h2-heading") %></h2>
+  <div id="uploaded-files-table-container">
+    <%= render partial: "uploaded_files", locals: { attachments: @form.legal_aid_application.attachments.bank_statement_evidence } %>
+  </div>
+
+  <%= form_with(
+        model: @form,
+        url: providers_legal_aid_application_bank_statements_path,
+        method: :patch,
+        local: true,
+      ) do |form| %>
+
+    <%= next_action_buttons(show_draft: true, form:) %>
+  <% end %>
+<% end %>
+
+<script nonce="<%= request.content_security_policy_nonce %>">
```

Comment:
clean code: would it be ideal to move this script from the view to an external JS file?